extensions/wakatime/src/today-summary.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSummary } from "./hooks";
+
+import { MenuBarExtra } from "@raycast/api";
+
+export default function Command() {
+  const summary = useSummary();
+
+  return (
+    <MenuBarExtra
+      icon="../assets/icon.png"
+      title={`${summary.data
+        ?.map(
```

Comment:
I'll prefer you evaluate this outside the prop. just a design preference of mine.